File: Im-system-sdk/src/lim/model/RequestBase.ts
export class RequestBase {
    appId:number;
    clientType: number;
    imei:string
    constructor(appId:number,clientType: number, imei: string) {
        this.appId = appId;
        this.clientType = clientType;
        this.imei = imei;
    }
}

File: common/src/main/java/com/itsyx/im/common/model/RequestBase.java
package com.itsyx.im.common.model;

import lombok.Data;

/**
 * @description:
 * @author: syx
 * @version: 1.0
 */
@Data
public class RequestBase {
    private Integer appId;

    private String operater;

    private Integer clientType;

    private String imei;
}
